index.html

```diff
@@ -83,5 +123,60 @@ <h2>Headliners</h2>
             </div>
         </section>
     </main>
+
+    <section id="about" class="unactive">
+        <div id="about-header">
+            <h2 id="about-title">All Points East <br />Fest 2023</h2>
+            <hr />
+            <p id="about-text">
+                The first year of All Points East was held in 2018 with LCD Soundsystem,
+                The xx and Björk headlining the festival weekend and Catfish and the Bottlemen,
+                The National and Nick Cave headlining the three standalone shows the following weekend.
+            </p>
+            <p id="about-contact">
+                Please contact us per Email for nay further questions about
+                All Points East 2023!<br />
+                <address>allpointseast2023@gmail.com</address>
+            </p>
+        </div>
+
+        <div id="see-past">
+            <h2>See the past All Points East Festivals</h2>
+            <hr />
+            <p>Take a look at the past two All Points East Festivals</p>
+            <div id="past-ctr">
+                <div class="past-event-card">
+                    <div class="overlay"></div>
+                    <h3>2019</h3>
+                </div>
+                <div class="past-event-card">
+                    <div class="overlay"></div>
+                    <h3>2021</h3>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <footer>
+        <div id="partners-ctr">
+            <h2>Partners</h2>
+            <hr />
+            <div id="partners">
+                <img src="./images/partners/mozilla.svg" />
+                <img src="./images/partners/googlekorea.svg" />
+                <img src="./images/partners/naver.svg" />
+                <img src="./images/partners/daumkakaologo.svg" />
+                <img src="./images/partners/airbnb.svg" />
+            </div>
+        </div>
+        <div id="inner-footer">
+            <img src="./images/logo.jpg" />
+            <small>
+                2023 All East Points.<br/>
+                Some rights reserved.
+            </small>
+        </div>
```

Comment:
- [ ] This is supposed to be a separate file. It is not supposed to be inside another file. You're required to create two webpages - home.html and about.html. Kindly do just that.

- [ ] There's omission of one section here. Look at the screenshot below to check the section that's missed on your about page.

### Your Design
![your-omit-section](https://github.com/Movgto/capstone_1_project/assets/26597432/bdb1ba2e-cbc6-4421-8cdb-125335320bb4)

### Template
![temp-omit-section](https://github.com/Movgto/capstone_1_project/assets/26597432/8f042245-cb91-4a80-9988-229ef78fbd0d)